x-pack/plugins/cases/server/services/alerts/index.ts

```diff
@@ -19,20 +17,6 @@ import {
   STATUS_VALUES,
 } from '../../../../rule_registry/common/technical_rule_data_field_names';
 
-export type AlertServiceContract = PublicMethodsOf<AlertService>;
```

Comment:
Keeping this similar to what we have done for other services. The mocks handle the `PublicMethodsOf` wrapping. If we external plugins need a contract in the future we can add it back and to all services.